src/cmd-oscontainer

```diff
@@ -42,7 +42,7 @@ def oscontainer_extract(containers_storage, src, dest,
     if commit is None:
         raise SystemExit("Failed to find label '{}'".format(OSCONTAINER_COMMIT_LABEL))
     iid = inspect['Id']
-    print(f"Preparing to extract cid: {iid}")
```

Comment:
hmm - we are running with python3 in el7 right?

Reply:
See my commit message...

TL;DR - we mangle this script to use Python2 on EL7

Reply:
yeah I found this: 
https://github.com/coreos/coreos-assembler/blob/460741e125f7f024926bda75b0b18808718da60b/build.sh#L100

Reply:
Also, using `.format()` is one of the original guidelines noted in #271 :wink:

Reply:
> Also, using `.format()` is one of the original guidelines noted in #271 wink

yep. that also says `    * target 3.4` :stuck_out_tongue_winking_eye:

Reply:
> yep. that also says  ` * target 3.4` 

It's not a bad target  :wink: